source/simulator/assemblers/stokes.cc

```diff
@@ -134,29 +139,146 @@ namespace aspect
                     {
                       data.local_matrix(i, j) += ((2.0 * eta * (scratch.grads_phi_u[i]
                                                                 * scratch.grads_phi_u[j]))
-                                                  + one_over_eta * pressure_scaling
-                                                  * pressure_scaling
-                                                  * (scratch.phi_p[i]
-                                                     * scratch.phi_p[j]))
+                                                 )
                                                  * JxW;
                     }
 
+
             }
-          else
+          if (this->get_parameters().use_bfbt)
             {
+              const double sqrt_eta=sqrt(eta);
```

Comment:
I see you prefer to not have spaces around assignments or comparisons. However, most of our code uses spaces. We sometimes allow both forms for loop counters (`unsigned int i=0; i<x; ++i`) but usually we have space around assignment and comparisons to make the code more readable. Please check your PR for other instances.